test/Microsoft.Azure.Devices.Edge.Test.Common/linux/PackageManagement.cs

```diff
@@ -62,7 +62,35 @@ public string[] GetInstallCommandsFromLocal(string path)
 
         public string[] GetInstallCommandsFromMicrosoftProd(Option<Uri> proxy)
         {
-            throw new NotImplementedException("aziot-edge and aziot-identity-service currently aren't available in package repos");
+            // we really support only two options for now.
+            string repository = this.os.ToLower() switch
+            {
+                "ubuntu" => $"https://packages.microsoft.com/config/ubuntu/18.04/multiarch/prod.list",
+                "debian" => $"https://packages.microsoft.com/config/debian/stretch/multiarch/prod.list",
+                _ => throw new NotImplementedException($"Don't know how to install daemon for '{this.os}'"),
+            };
+
+            return this.packageExtension switch
+            {
+                SupportedPackageExtension.Deb => new[]
+                {
+                    $"curl {repository} > /etc/apt/sources.list.d/microsoft-prod.list",
+                    "curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /etc/apt/trusted.gpg.d/microsoft.gpg",
+                    $"apt-get update",
```

Comment:
No need for the $ for these strings, right? Same for some of the yum commands below.